### ISSUE TITLE
Event for raids from certain uncolonized provinces

### ISSUE THREAD

**Issue by PhelanBavaria** [PhelanBavaria/ancienttimeline#18]
Addition of an event or multiple to get raided from uncolonized provinzes.
The raids should probably occur if you border any uncolonized Celtic, Germanic, Berber, etc. provinces or perhaps just all uncolonized provinces.
The result could be:
- rebels that demand money if they succeed
- a province modifier (there is a similar event in ET I think)
- change of the province's development

**Comment by Firesoul7** [comment]
Hmm... Perhaps, but I'm not big on 100 kinds of rebels. I'd much rather that nations are created in these zones, and that these nations are made aggressive and powerful in order to raid. There could be a new CB. 

Sent from my Samsung device

-------- Original message --------

**Comment by PhelanBavaria** [comment]
The issue with that is that they probably would take land too, there is no way, at least not to my knowledge, to restrict them of only taking money. Also, raids happened fast, they didn't have to siege down every fort in the area to get the money.
Another issue is that raids happened before you could really speak of a sophisticated tribe or country. Many of these raiders were just a little tribe of the size of an extended family.

**Comment by qweytr** [comment]
I agree that a new raider rebel type could be added, but it shouldn't be the only way to simulate raiding.  There should also be an event similar to the horde plunder events. When a Celtic or Germanic country occupies the province, the occupier gains some money and prestige, while the owner loses some. I don't think it's a problem that these countries could take some land in the raid wars and only provinces with forts are hard to siege. Other provinces can be sieged and raided quickly. Provinces with forts should be hard to raid.

**Comment by Firesoul7** [comment]
Fully agreed with Qweytr.

**Comment by PhelanBavaria** [comment]
Yes, there should be another event as you said.
What I meant is that, if a nation declares war to raid, then they have to siege down provinces, especially those with forts, to gain any warscore and to be able to demand anything, which I find quite silly since raids happened quickly, they didn't need to take down all forts in the area before they could get the money.

**Comment by qweytr** [comment]
The point of the raid wars would not be to accumulate warscore and demand stuff. It would be to run though the unfortified provinces, siege them quickly and take the gold from the plunder events.

**Comment by PhelanBavaria** [comment]
I see, but still you have to gain warscore to end the war, and if you are a little tribe who attacks Carthage while they don't have their army there, then chances are they will flatten you and just demand all the gold back that you gained from the raid, no?

**Comment by qweytr** [comment]
You're right. That would happen. But if there was a small annoying tribe of raiders next to Carthage, they would conquer it anyway if they could. The only problem is that it is currently way too easy to maintain control over peripheral regions. I think that that is the kind of raiding that could be left to be represented by the rebels.
